backend/events/apps.py

```diff
@@ -85,6 +85,24 @@ def on_tuples(cc_event, block_number, tx_id, tx_status):
                 queue=worker_queue
             )
 
+        if 'ComputePlans' in payload:
+            handle_compute_plan_events(payload['ComputePlans'])
+
+
+def handle_compute_plan_events(cps):
+    for cp in cps:
+        handle_compute_plan_event(cp)
+
+
+def handle_compute_plan_event(cp):
+    cpID = cp['computePlanID']
```

Comment:
Python variable should be defined in lower case with underscore (https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

For reserved keywords like id, you could add a suffix `_` (https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments).

Reply:
Good point!

Reply:
Fixed: https://github.com/SubstraFoundation/substra-backend/pull/90/commits/bb2b5ba2928b13420a71cf7a94268225934ff8f0